src/bilder/images/edxapp/templates/lms_only.yml

```diff
@@ -0,0 +1,114 @@
+ACCOUNT_MICROFRONTEND_URL: null
+ACE_CHANNEL_DEFAULT_EMAIL: django_email
+############################
+# We aren't using Sailthru #
+############################
+# ACE_CHANNEL_SAILTHRU_API_KEY: ''
+# ACE_CHANNEL_SAILTHRU_API_SECRET: ''
+# ACE_CHANNEL_SAILTHRU_DEBUG: true
+# ACE_CHANNEL_SAILTHRU_TEMPLATE_NAME: null
+ACE_CHANNEL_TRANSACTIONAL_EMAIL: django_email
+ACE_ENABLED_CHANNELS:
+- django_email
+ACE_ENABLED_POLICIES:
+- bulk_email_optout
+ACE_ROUTING_KEY: edx.lms.core.default
+# ALTERNATE_WORKER_QUEUES: cms  # The code already has this default
+ANALYTICS_API_KEY: ''
+ANALYTICS_API_URL: http://localhost:18100
+API_ACCESS_FROM_EMAIL: api-requests@example.com
+API_ACCESS_MANAGER_EMAIL: api-access@example.com
+API_DOCUMENTATION_URL: http://course-catalog-api-guide.readthedocs.io/en/latest/
+AUDIT_CERT_CUTOFF_DATE: null
+AUTH_DOCUMENTATION_URL: http://course-catalog-api-guide.readthedocs.io/en/latest/authentication/index.html
+BULK_EMAIL_ROUTING_KEY_SMALL_JOBS: edx.lms.core.default
+CONTACT_MAILING_ADDRESS: SET-ME-PLEASE
+CREDIT_HELP_LINK_URL: ''
+DCS_SESSION_COOKIE_SAMESITE: Lax
+DCS_SESSION_COOKIE_SAMESITE_FORCE_ALL: true
+EDXNOTES_INTERNAL_API: http://localhost:18120/api/v1
+EDXNOTES_PUBLIC_API: http://localhost:18120/api/v1
+# EDX_API_KEY: PUT_YOUR_API_KEY_HERE  # DEPRECATED SETTING
+########################################################
+# We are not using any of the enterprise functionality #
+########################################################
+# ENTERPRISE_COURSE_ENROLLMENT_AUDIT_MODES:
+# - audit
+# - honor
+# ENTERPRISE_CUSTOMER_SUCCESS_EMAIL: customersuccess@edx.org
+# ENTERPRISE_ENROLLMENT_API_URL: https://localhost:18000/api/enrollment/v1/
+# ENTERPRISE_INTEGRATIONS_EMAIL: enterprise-integrations@edx.org
+# ENTERPRISE_SUPPORT_URL: ''
+# ENTERPRISE_TAGLINE: ''
+GIT_REPO_DIR: /edx/var/edxapp/course_repos
+GOOGLE_ANALYTICS_LINKEDIN: ''
+GOOGLE_ANALYTICS_TRACKING_ID: ''
+GOOGLE_SITE_VERIFICATION_ID: ''
+HTTPS: 'on'
+LEARNING_MICROFRONTEND_URL: null
+LTI_AGGREGATE_SCORE_PASSBACK_DELAY: 900
+LTI_USER_EMAIL_DOMAIN: lti.example.com
+MAILCHIMP_NEW_USER_LIST_ID: null
+OAUTH_DELETE_EXPIRED: true
+OAUTH_ENFORCE_SECURE: true
+OAUTH_EXPIRE_CONFIDENTIAL_CLIENT_DAYS: 365
+OAUTH_EXPIRE_PUBLIC_CLIENT_DAYS: 30
+OPTIMIZELY_PROJECT_ID: null
+ORDER_HISTORY_MICROFRONTEND_URL: null
+ORGANIZATIONS_AUTOCREATE: true
+PAID_COURSE_REGISTRATION_CURRENCY:
+- usd
+- $
+PARENTAL_CONSENT_AGE_LIMIT: 13
+PDF_RECEIPT_BILLING_ADDRESS: 'Enter your receipt billing
+
+    address here.
+
+    '
+PDF_RECEIPT_COBRAND_LOGO_PATH: ''
+PDF_RECEIPT_DISCLAIMER_TEXT: 'ENTER YOUR RECEIPT DISCLAIMER TEXT HERE.
+
+    '
+PDF_RECEIPT_FOOTER_TEXT: 'Enter your receipt footer text here.
+
+    '
+PDF_RECEIPT_LOGO_PATH: ''
+PDF_RECEIPT_TAX_ID: 00-0000000
+PDF_RECEIPT_TAX_ID_LABEL: fake Tax ID
+PDF_RECEIPT_TERMS_AND_CONDITIONS: 'Enter your receipt terms and conditions here.
+
+    '
```

Comment:
Are any of the `PDF_...` needed?

Reply:
Eventually, most likely, but the focus for this is to get the baseline system set up and then we can fine-tune the fiddly bits as we go.